src/transformers/generation/logits_process.py

```diff
@@ -1191,9 +1196,30 @@ def _validate_arguments(self):
                 f"Each key in `sequence_bias` has to be a non-empty tuple of positive integers, but is "
                 f"{sequence_bias}."
             )
-        if any(not isinstance(bias, float) for bias in sequence_bias.values()):
+        if isinstance(sequence_bias, list) and any(
+            any(
+                not (
+                    isinstance(token_bias_pair, list)
+                    and all(isinstance(token_id, int) and token_id > 0 for token_id in token_bias_pair)
+                    or isinstance(token_bias_pair, float)
+                )
+                for token_bias_pair in sequence
+            )
+            or len(sequence) == 0
+            for sequence in sequence_bias
+        ):
+            raise ValueError(
+                f"Each element in `sequence_bias` has to be a non-empty list of lists of positive integers and float, but is "
+                f"{sequence_bias}."
+            )
+        if isinstance(sequence_bias, dict) and any(not isinstance(bias, float) for bias in sequence_bias.values()):
             raise ValueError(f"`sequence_bias` has to be a dict with floats as values, but is {sequence_bias}.")
 
+    def _convert_list_arguments_into_dict(self):
```

Comment:
```suggestion
    def _convert_list_arguments_into_dict(self):
        """BC: we used to accept `dict{tuple of tokens: float}`"""
```
This will help us track the effort to keep backwards compatibility